examples/06_benchmarks/benchmark_utils.py

```diff
@@ -223,15 +225,21 @@ def recommend_k_fastai(model, test, train, top_k=DEFAULT_K, remove_seen=True):
     return topk_scores, t
 
 
-def prepare_training_ncf(train, test):
+def prepare_training_ncf(df_train, df_test):
+    df_train.sort_values(["userID"], axis=0, ascending=[True], inplace=True)
+    df_test.sort_values(["userID"], axis=0, ascending=[True], inplace=True)
+    train = "./df_train.csv"
+    test = "./df_test.csv"
```

Comment:
I think here it should be better to add a temporary path, so it is erased automatically. Something like:
```suggestion
    tmp_dir = TemporaryDirectory()
    train = os.path.join(tmp_dir.name, "df_train.csv")
    test = os.path.join(tmp_dir.name, "df_test.csv")
```